timing-attack/timing-attack.ts

```
@@ -0,0 +1,27 @@
+function timingAttackNoncompliant() {
+  app.get("/user/login", function (req: Request, res: Response) {
+    // Noncompliant: '===' operator is used with sensitive data field. 
+    if (password === "myPass") {
+      // logIn()
+    }
+  });
+}
+// {/fact}
+
+// {fact rule=timing-attack@v1.0 defects=0}
+import express, {Request, Response} from 'express'
```

Biome (2.1.2)

[error] Shouldn't redeclare 'express'. Consider to delete it or rename it.

'express' is defined here:


(lint/suspicious/noRedeclare)